### DISCUSSION TITLE
blog_11ty/posts/Others/2024-07-30-%E6%92%B8%E4%B8%80%E4%B8%AA%E7%BA%AF%E5%89%8D%E7%AB%AF%E5%AE%9E%E7%8E%B0%E7%9A%84FFmpeg%E8%A7%86%E9%A2%91%E8%BD%AC%E5%8E%8B%E7%BC%A9%E5%B7%A5%E5%85%B7/

### DISCUSSION THREAD

**Discussion by giscus** [joisun/blog_11ty#8] [General]
# blog_11ty/posts/Others/2024-07-30-%E6%92%B8%E4%B8%80%E4%B8%AA%E7%BA%AF%E5%89%8D%E7%AB%AF%E5%AE%9E%E7%8E%B0%E7%9A%84FFmpeg%E8%A7%86%E9%A2%91%E8%BD%AC%E5%8E%8B%E7%BC%A9%E5%B7%A5%E5%85%B7/

simple style blog site

https://joisun.github.io/blog_11ty/posts/Others/2024-07-30-%E6%92%B8%E4%B8%80%E4%B8%AA%E7%BA%AF%E5%89%8D%E7%AB%AF%E5%AE%9E%E7%8E%B0%E7%9A%84FFmpeg%E8%A7%86%E9%A2%91%E8%BD%AC%E5%8E%8B%E7%BC%A9%E5%B7%A5%E5%85%B7/

**Comment by joisun** [comment]
test
